cpu/stm32/include/periph/cpu_gpio_ll.h

```diff
@@ -46,6 +46,24 @@ typedef enum {
     GPIO_DRIVE_STRONGEST = 0
 } gpio_drive_strength_t;
 
+#if defined(GPIO_OSPEEDR_OSPEED0) || defined(GPIO_OSPEEDER_OSPEEDR0) \
+    || defined(GPIO_OSPEEDER_OSPEED0) || defined(GPIO_OSPEEDR_OSPEEDR0)
+/* Modern STM32 GPIO config registers with the OSPEEDR register support full
+ * 4 slew rates, legacy STM32F1 style only have three slew rates. We define
+ * slow and fast to the same value, so that we have three options:
+ * 1. SLOWEST: 2 MHZ
+ * 2. SLOW/FAST: 10 MHZ
+ * 3. FASTEST: 50 MHz
+ */
+#define HAVE_GPIO_SLEW_T
+typedef enum {
+    GPIO_SLEW_SLOWEST = 0,
+    GPIO_SLEW_SLOW = 1,
+    GPIO_SLEW_FAST = 1,
```

Comment:
In GD32V driver, I defined `GPIO_SLEW_FAST = 2`. Was that wrong?

Reply:
No, it is up to the programmer to match the fuzzy words `SLOWEST`, `SLOW`, ... to the speeds provided by the MCU. I don't thing there is any right or wrong, as those fuzzy words are subjective.

I'll update this to match the GD32V, though. I think it is better to be consistent here when the peripheral is basically the same.